tests/integration/buildcmd/test_build_cmd.py

```diff
@@ -1585,8 +1588,59 @@ def test_nested_build(self, use_container, cached, parallel):
             )
 
 
+class TestBuildWithNestedStacks3Level(NestedBuildIntegBase):
+    """
+    In this template, it has the same structure as .aws-sam/build
+    build
+        - template.yaml
+        - FunctionA
+        - ChildStackX
+            - template.yaml
+            - FunctionB
+            - ChildStackY
+                - template.yaml
+                - FunctionA
+                - MyLayerVersion
+    """
+
+    template = os.path.join("deep-nested", "template.yaml")
+
+    # @pytest.mark.flaky(reruns=3)
```

Comment:
was this test supposed to be flaky? if not lets remove this comment.

Reply:
good catch! I commented out it for local testing purpose, let me bring it back! thx